.github/workflows/ci.yml

```diff
@@ -100,6 +100,29 @@ jobs:
       - name: Check unused dependencies
         run: cargo machete
 
+  check-msrv:
```

Comment:
Am I missing how this is checking MSRV? It looks like all it does is run `carrgo test`, same as the below workflow.